### PR TITLE
Renaming + Certora adjustments

### PR DESCRIPTION
**Pull request by sunbreak1211**

### CODE REVIEW

File: test/DaiUsds.t.sol [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        address receiver = address(123);
+        assertEq(dai.balanceOf(receiver),                  0);
+        assertEq(usds.balanceOf(receiver),                 0);
+
+        dai.approve(address(daiUsds), 1_500 * WAD);
+        vm.expectEmit(true, true, true, true);
+        emit DaiToUsds(address(this), receiver,  1_500 * WAD);
+        daiUsds.daiToUsds(receiver, 1_500 * WAD);
+        assertEq(dai.balanceOf(address(this)),   6_500 * WAD);
+        assertEq(dai.balanceOf(receiver),                 0);
```
Comment (telome) [inline test/DaiUsds.t.sol:79]:
```suggestion
        assertEq(dai.balanceOf(receiver),                  0);
```

File: test/Usds.t.sol [hunk 2]
```diff
@@ -39,34 +39,34 @@ contract Nst2 is UUPSUpgradeable {
...
-        _symbol_ = "NST";
+        usds = Usds(address(new ERC1967Proxy(imp, abi.encodeCall(Usds.initialize, ()))));
+        assertEq(usds.version(), "1");
+        assertEq(usds.wards(address(this)), 1);
+        assertEq(usds.getImplementation(), imp);
+
+        _token_ = address(usds);
+        _contractName_ = "Usds";
+        _tokenName_ = "Sky USD";
+        _symbol_ = "USDS";
```
Comment (oldchili) [inline test/Usds.t.sol:62]:
```suggestion
        _tokenName_ = "USDS Stablecoin";
        _symbol_ = "USDS";
```

File: test/Usds.t.sol [hunk 3]
```diff
@@ -39,34 +39,34 @@ contract Nst2 is UUPSUpgradeable {
...
     }
 
     function invariantMetadata() public view {
-        assertEq(nst.name(), "Nst Stablecoin");
-        assertEq(nst.symbol(), "NST");
-        assertEq(nst.version(), "1");
-        assertEq(nst.decimals(), 18);
+        assertEq(usds.name(), "Sky USD");
```
Comment (oldchili) [inline test/Usds.t.sol:66]:
```suggestion
        assertEq(usds.name(), "USDS Stablecoin");
```

**Review by telome — APPROVED** [review]
LGTM

File: test/Usds.t.sol (hunk 3 above)
Comment (sunbreak1211) [reply test/Usds.t.sol]: Yeah just realized, fixed!

File: test/Usds.t.sol (hunk 2 above)
Comment (sunbreak1211) [reply test/Usds.t.sol]: Yeah just realized, fixed!

**Review by oldchili — APPROVED** [review]
LGTM

**Pull request merged**